app/src/main/java/com/lambdaschool/hackathon_portal/ui/fragments/dashboard/DashboardFragment.kt

```diff
@@ -89,6 +87,11 @@ class DashboardFragment : Fragment() {
         override fun onBindViewHolder(holder: ViewHolder, position: Int) {
             val data = userHackathons[position]
             holder.nameView.text = data.hackathon_name
+            holder.itemView.setOnClickListener {
+                val bundle = Bundle()
+                bundle.putInt("hackathon_id", data.hackathon_id)
+                findNavController().navigate(R.id.detailsFragment, bundle)
```

Comment:
Injecting the NavController would be better performance wise.

adding to line 37
```
@Inject
lateinit var navController: NavController
```
```suggestion
                navController.navigate(R.id.detailsFragment, bundle)
```

Reply:
Changed in commit 770119ff532eda3f5b007d568d5ecbf612792a90